Tests/SessionTests.swift

```diff
@@ -937,7 +945,7 @@ class SessionTestCase: BaseTestCase {
         XCTAssertEqual(response?.result.isSuccess, true)
         XCTAssertTrue(session.requestTaskMap.isEmpty)
     }
-    // TODO: Confirm retry logic.
```

Comment:
I confirmed that everything looks correct with this test so I removed the comment.